tools/census_contrib/src/census_contrib/metadata.py

```diff
@@ -140,6 +142,13 @@ def validate_metadata(args: Arguments, metadata: EmbeddingMetadata) -> Embedding
             "Metadata: description must be string between 1 and {MAX_DESCRIPTION_LENGTH} characters in length",
         )
 
+    # 7. Name must have length < 24 characters
+    MAX_NAME_LENGTH = 24
```

Comment:
I figured this could be a sensible value.

Reply:
any reason to have it so short?  Unless there is an external dependency on 24, I'd go for something that can handle a long project name (e,g, < 128)

Reply:
Nah, we can leave it to 128.